templates/looking-for-helpers.md

```diff
@@ -0,0 +1,31 @@
+This was part of a SNIC training newsletter. Saving it so that we can reuse
```

Comment:
```suggestion
# Looking for helpers

This was part of a SNIC training newsletter. Saving it so that we can reuse
```
This should fix the sphinx warning